test/unit/test_utils.js

```diff
@@ -116,6 +126,10 @@ class XRefMock {
   async fetchIfRefAsync(obj) {
     return this.fetchIfRef(obj);
   }
+
+  addEntry(ref, obj) {
+    this._map[ref.toString()] = obj;
+  }
```

Comment:
This "full" `XRef` class no longer has such a method, so this one should probably be removed as well?